pinot-controller/src/main/java/org/apache/pinot/controller/util/TableSizeReader.java

```diff
@@ -225,14 +228,13 @@ public TableSubTypeSizeDetails getTableSubtypeSize(String tableNameWithType, int
           .setValueOfTableGauge(tableNameWithType, ControllerGauge.TABLE_STORAGE_EST_MISSING_SEGMENT_PERCENT,
               missingPercent);
       if (subTypeSizeDetails.missingSegments == numSegments) {
-        LOGGER.warn("Failed to get size report for all {} segments: {} for table: {}", numSegments, missingSegments,
-            tableNameWithType);
+        LOGGER.warn("Failed to get size report for all {} segments of table: {}", numSegments, tableNameWithType);
         subTypeSizeDetails.reportedSizeInBytes = -1;
         subTypeSizeDetails.estimatedSizeInBytes = -1;
       } else {
-        LOGGER
-            .warn("Missing size report for {} out of {} segments: {} for table {}", subTypeSizeDetails.missingSegments,
-                numSegments, missingSegments, tableNameWithType);
+        LOGGER.warn("Missing size report for {} out of {} segments for table {}. Listing up to {} segments: {}",
```

Comment:
I am not sure if we really care much about the specific segment names. We can just list how many were missed.

Reply:
Changed.